server/node.go

```diff
@@ -116,7 +117,7 @@ func BootstrapCluster(clusterID string, eng engine.Engine) (*client.KV, error) {
 	// Create a KV DB with a local sender.
 	lSender := kv.NewLocalSender()
 	localDB := client.NewKV(kv.NewTxnCoordSender(lSender, clock), nil)
-	s := storage.NewStore(clock, eng, localDB, nil)
+	s := storage.NewStore(clock, eng, localDB, nil, multiraft.NewLocalRPCTransport())
```

Comment:
In the case of the BootstrapCluster function, it is always called in the context of a single-node cluster so it's fine to hard-code LocalRPCTransport here (just like we use NewLocalSender two lines up and pass nil as the Gossip parameter to NewStore.

Elsewhere, the real transport depends on a Gossip object, which is nil in many of our tests. We could make a dummy gossip object in all of those tests, which might theoretically give us a more complete test but I think it's probably not worth it.